test/organisationSpec.js

```diff
@@ -82,22 +84,23 @@ describe('Organisation API interaction test', function() {
   });
 
   it('should handle errors while creating a new organisation', function() {
-    var api = new its.Sdk({
+    var api = new connection.Connection({
       authPrincipal: 'principal',
       authPassword: 'secret'
     });
     var organisation = new its.Organisation('1');
+    organisation.connection = api;
     var cb = jasmine.createSpy('callback');
     var ecb = jasmine.createSpy('callback');
 
-    var output = api.createOrganisation(organisation, cb, ecb);
+    var output = organisation.createOrganisation(cb, ecb);
 
     var request = jasmine.Ajax.requests.mostRecent();
     var url = 'https://api.itslanguage.nl/organisations';
     expect(request.url).toBe(url);
     expect(request.method).toBe('POST');
     var expected = {id: '1'};
-    expect(request.data()).toEqual(expected);
+    expect(_.isEqual(request.data(), expected));
```

Comment:
yes it is. It's a deep equality check by underscore. It will fail with the Jasmine isEqual.